src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 16px;
+
+  &__product {
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-top: 16px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin-top: 40px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-1;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-2;
+    margin-top: 4px;
+  }
+
+  &__review {
+    margin-top: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-count {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-1;
+  }
+
+  &__price {
+    margin-top: 24px;
+    line-height: 18px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      color: $text-color-2;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      color: $text-color-1;
+    }
+  }
+
+  &__button-buy {
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    height: 40px;
+    background-color: $button-color;
+    text-decoration: none;
+    color: #fff;
+    border-radius: 5px;
+    border: 1px solid transparent;
+
+    transition: 0.2s;
```

Comment:
It should be specified what exactly is changing  and transition-timing-function. For example `transition: color 0.2s ease`. In this way, you need to list everything that changes on hover and what you want to smoothly change on transition